packages/utils/src/index.ts

```diff
@@ -1 +1,2 @@
 export * from './liquidity.js';
+export * from './bitcoin/blockstream.js';
```

Comment:
add an `index.ts` file in `bitcoin` folder and then `export * from './blockstream.js`

Reply:
@saadjhk FYI: there is no index.ts for the substrate folder